cyber-resilience-sig/coordination/enisa/deliverable-2-6.md

```diff
@@ -0,0 +1,48 @@
+
+## Context
+
+Provide feedback to the [EU Commission][]'s [open call for evidence](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act_en) as part of its [initiative to revise the Cybersecurity Act (CSA)](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act). Urge for a stronger technical and strategic role for ENISA in light of the [recent funding issues of the US CVE program](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#2025_funding_issues) and the increased importance of vulnerability handling in EU legislation.
+
+## Goals
+
+* Urge the EU Commission to increase ENISA's mandate regarding CVE handling
+* Avoid disproportionate reliance on U.S. government funding for CVEs
+
+## Feedback
+
+In light of the [CSA review](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act) and wider need for harmonisation of vulnerability disclosure requirements across several EU regulations (e.g. CRA, NIS2, DORA, AI act) as well as further sharing of supply-chain best-practices, we wish to share and reiterate the importance of the Common Vulnerabilities and Exposures (CVE) system and a stronger technical and strategic role for ENISA given its growing international objective:
+
+* We recognise the importance of increasing EU resources to ENISA as part of CSA review. Notably, ENISA’s role as a CVE Numbering Authority (CNA) is important and industry welcomes ENISA’s goal to become a Root CNA.
+* MITRE’s role as Secretariat plays a key role to provide a resource to boost supply chain resilience.  Whilst it is regrettable the recent publicity and uncertainty in funding we are pleased that this matter has now been partially resolved.  
+* The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance and need for wider  global support. The establishment of [the CVE foundation](https://www.thecvefoundation.org/) provides ever greater visibility in supporting this crucial namespace pillar of global supply chain resilience.
+* Indeed, this has also been recognised by several European governments who subsequently confirmed their wish to support (technically and financially) and thus we would hope that this support can be mirrored at an EU level, perhaps via ENISA, into a system which is mirrored infrastructure in Europe and co-developed.
+* We also see the growing role of national Computer Security Incident Response Team (CSIRTs), such as the National Cyber Security Centre (NSSC) in Ireland, largely supporting bug bounty programs, Capture the Flag (CTF) and other activities with local academia, government and businesses, investing in vulnerability discovery and management, and thus important to double down on the CVE program in a uniformed, cross-European and international way.
```

Comment:
It seems like that should be "NCSC Ireland", as indicated at https://www.ncsc.gov.ie/ too.

Reply:
Is the typo fix all you had in mind here?